quandary.py

```diff
@@ -557,35 +560,28 @@ def __dump(self, *, pcof0=[], runtype="simulation", datadir="./run_dir"):
         # If not standard Hamiltonian model, write provided Hamiltonians to a file
         if not self.standardmodel:
             # Write non-standard Hamiltonians to file  
-            self._hamiltonian_filename= "hamiltonian.dat"
-            with open(os.path.join(datadir, self._hamiltonian_filename), "w", newline='\n') as f:
+            # Write system Hamiltonian (complex)
+            self._hamiltonian_filename_Hsys= "hamiltonian_Hsys.dat"
+            H = self.Hsys
+            with open(os.path.join(datadir, self._hamiltonian_filename_Hsys), "w", newline='\n') as f:
                 f.write("# Hsys_real \n")
```

Comment:
I think this should be updated for the new format, maybe
```suggestion
                f.write("# row col Hsys_real Hsys_imag\n")
```